### PR TITLE
Do not reset the color map ranges on I/O transform

### LINKED ISSUE #528: When applying a transform to the image in the I/O widget, the colormap ranges get reset. This should be disabled.

### PR DESCRIPTION
**Pull request by bnmajor**
Fixes #528

### CODE REVIEW

**Comment by bnmajor** [comment]
This branch now respects any color map range that is set for the entire LLNL I/O workflow - the range should not be automatically reset by any selections or changes until the workflow has been completed by selecting the `Complete and Reload Instrument` option.

File: hexrd/ui/main_window.py [hunk 1]
```diff
@@ -85,7 +85,8 @@ def __init__(self, parent=None, image_files=None):
         self.load_widget.ui.setVisible(False)
         self.workflow_widgets[WORKFLOW_HEDM].append(self.load_widget.ui)
 
-        self.import_data_widget = ImportDataPanel(self.ui)
+        self.import_data_widget = ImportDataPanel(self.color_map_editor,
```
Comment (cjh1) [inline hexrd/ui/main_window.py:88]: Its not great to have the import panel depend on the color map editor. I wonder if there is another way, may be with signal connected together here? May be the import data widget could emit an 'updating started' and 'complete signal' which could connected by some function here to enable/disable the color map editor. Lets think about it a bit.

Comment (joelvbernier) [reply hexrd/ui/main_window.py:88]: Ok, we can hold on this for a bit to figure a better implementation.

**Comment by joelvbernier** [comment]
Should we merge this up, or rework?

**Comment by cjh1** [comment]
@joelvbernier We can merge and rework, I will raise a issue for it. @bnmajor can you rebase on master, thanks!

**Comment by bnmajor** [comment]
@cjh1 Sure thing!

**Review by cjh1 — APPROVED** [review]
I have raise #547 for a future rework.

**Pull request merged**